ui-ngx/src/app/modules/home/components/widget/lib/gateway/gateway-widget.models.ts

```diff
@@ -181,6 +181,8 @@ export interface ConnectorBaseConfig {
   server?: ServerConfig;
   broker?: BrokerConfig;
   workers?: WorkersConfig;
+  master?: ModbusMasterConfig;
+  slave?: ModbusSlave;
```

Comment:
It may be better to rewrite it as a union type with separate types for different connector protocols